examples/experimental/dagster-airlift/examples/many-dags/scripts/generate_yaml_files.py

```diff
@@ -0,0 +1,32 @@
+import argparse
```

Comment:
yeah would rather have this managed in memory rather than having to codegen these files

Reply:
I think the codegen is important actually. Worth profiling how long it takes to read in the yaml potentially.